vela: add page (#4942)
tldr-pages/tldr: pages/common/vela.md

# vela

> > Command line tools for Vela pipeline.

Reviewer: Perhaps this will read better?
```suggestion
> Command line tools for the Vela pipeline.
```

> More information: <https://go-vela.github.io/docs/cli/>.

> - Trigger pipeline to run from git tagged version:

Reviewer 2: ```suggestion
- Trigger a pipeline to run from a git branch, commit or tag:
```

> `vela add deployment --org {{organization}} --repo {{repository_name}} --target {{environment}} --ref {{branch, commit, or refs/tags/git_tag}} --description "{{deploy_description}}"`

Reviewer 2: ```suggestion
`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{environment}} --ref {{branch|commit|refs/tags/git_tag}} --description "{{deploy_description}}"`
```

> - Get list of deployments for a repo:

Reviewer 2: ```suggestion
- List deployments for a repository:
```

# vela

> > Command line tools for vela pipeline.

Reviewer 2: ```suggestion
> Command line tools for Vela pipeline.
```

> > More information: <https://go-vela.github.io/docs/>.

Reviewer 2: ```suggestion
> More information: <https://go-vela.github.io/docs/cli/>.
```

- Trigger pipeline to run from git tagged version:

> `vela add deployment --org {{git org}} --repo {{repo name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git tag}} --description "{{Deploy description}}"`

Reviewer 2: We use `snake_case` for variables:
```suggestion
`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`
```
The `--org` which values could have? `github`?
What is a `git org`?

> `vela add deployment --org {{organization}} --repo {{repository_name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`

Reviewer 2: Can you add more examples? Maybe 1 for github org, another for prod environment, Can I deploy using a branch?
```suggestion
`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{environment}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`
```
Also Vela has more subcommands like `view` and `get`

Author: I can add examples for `view` and `get` (although I'll just copy from the docs page since I personally have not used those commands before).  I did just check, it is commit/branch/ or tag for the `--ref` option
